net-ftp/yafc/files/yafc-1.3.7-fix-autoreconf.patch

```diff
@@ -0,0 +1,15 @@
+Pascal Jäger <pascal.jaeger@leimstift.de> (2022-12-10)
```

Comment:
If you have an "upstream place" where you have taken please refers to it, for eg:

```
https://bla/bla/bla
Pascal Jäger <pascal.jaeger@leimstift.de> (2022-12-10)
```

Reply:
I have none, I came up with this after googling around a bit and reading docs